spec/support/unit/rails_application.rb

```diff
@@ -26,7 +26,7 @@ def create
     def load
       load_environment
 
-      if rails_version > 5 && bundle.includes?('actiontext')
+      if rails_version >= 6 && bundle.includes?('actiontext')
         add_action_text_migration
```

Comment:
But it can be used in versions below if included in the Gemfile that is why the other condition. `(bundle.includes?('actiontext'))`

For example: https://www.driftingruby.com/episodes/using-action-text-in-a-rails-5-2-application

Regardless, I see no problem with this adjustment because the actiontext is only tested on Rails 6.

Why not remove even the second condition?

And I think you can remove this line too: https://github.com/thoughtbot/shoulda-matchers/blob/master/gemfiles/rails_6_0.gemfile#L36

Reply:
>I notice now that I forgot to test the actiontext on other versions. It's only tested on Rails 6. If we decide to not test on 5.2, I see no problem with this adjustment.

Unless we use a patched fork to work around, `rails/actiontext` gem dependencies for `rails < 6` can not be met from official sources.

We can also use `bundle.includes?('actiontext')` only rather than checking the version.

Reply:
Yeah, you are right. We can ignore the version. I don't see any problem with this change, but I think we can do even better. Let me take another look at the project.

Reply:
Yes, maybe later it becomes a refactoring candidate as we do not rely on version but rather on definition of `actiontext`.